packages/shopify-codemod/transforms/utils.js

```diff
@@ -11,3 +11,14 @@ export function findLastMember(node) {
   }
   return node;
 }
+
+export function insertAfterDirectives(body, newNode) {
+  let i = 0;
+  for (;i < body.length; i++) {
```

Comment:
Ugh, hate this way of expressing this. Can we at least move the `let i = 0` in as the first condition?